docs/version-history/v24-version.hist

```diff
@@ -1,5 +1,9 @@
 *** 24.11.0 features
 
+- The `htcondor annex` tool now has (limited) support for AWS' EC2 annexes.
+  The ``condor_annex`` tool has been withdrawn.
+  :jira:`1630`
+  
```

Comment:
Do we want to same something explicitly about moving from `condor_annex` to `htcondor annex` ?

Reply:
Added.